notebooks/analyze_looping_across_lending_protocols.ipynb

```diff
@@ -206,21 +211,23 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Tokens to analyze\n",
+    "# Tokens\n",
     "tokens = [\"ETH\", \"wBTC\", \"USDC\", \"DAI\", \"USDT\", \"wstETH\", \"LORDS\", \"STRK\", \"UNO\", \"ZEND\"]\n",
     "\n",
     "# Initialize a dictionary to keep track of borrowed and collateral assets\n",
     "borrowed_assets = {}\n",
```

Comment:
The name doesn't really reflect that the values in the dict are users.

Reply:
Done! ✅